config.h

```diff
@@ -213,6 +213,19 @@
 
 #endif
 
+// If defined, enable a low batt icon triggered by a GPIO pin whose BCM number is given.
+// #define LOW_BATT_PIN 19
+
+#if defined(LOW_BATT_PIN)

```

Comment:
Let's not gate config items with an #ifdef (will just get ignored when not in use), that will make the config flow a bit more linear to read.